cockpit/image-builder-frontend.spec

```diff
@@ -0,0 +1,39 @@
+# XXX: package name?
+Name:           image-builder-frontend
```

Comment:
To be able to release it in time, I think we decided to keep the name `cockpit-composer`.